canonical/lxd-ui
# Limit number of ips displayed in the instance list to 5

## Done

- limit the number of ip addresses in the instance list to 5 with a show more link in case there are more

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check the instance list with an instance that has multiple ip addresses
